rocket_sqlite_oracle/src/main.rs

```diff
@@ -166,6 +173,7 @@ async fn bid_price_at_time(
     let res = db::USDTickerResponse {
         timestamp: row.timestamp.try_into().unwrap(),
         price: row.price.try_into().unwrap(),
+        units: row.units,
```

Comment:
I would have been ok with just hard-coding `millicents` as the return value here and not saving this information to the database.

Reply:
What I wanted to do was the sensible option of:
```
        select
            timestamp,
            ask_price as price,
            "millicents" as units,
            ask_signature as signature
        from signatures
```

but that is not allowed with rocket as the migrations expect to find the `units` column in the table data. This was done this way to enable inserting in other values as required should another service with different units be employed.